tests/framework/defs.py

```diff
@@ -28,3 +28,5 @@
 ENV_TEST_IMAGES_S3_BUCKET = 'TEST_MICROVM_IMAGES_S3_BUCKET'
 """Default test session root directory path"""
 DEFAULT_TEST_SESSION_ROOT_PATH = "/srv"
+"""Name of the performance test results folder"""
+PERF_TEST_RESULTS_DIR = "../test_artifacts"
```

Comment:
Name and/or description should reflect the fact that this is a relative path.

Also, for defining relative paths, we should set in stone one variable that fixes that path (e.g. `FC_WORKSPACE_DIR`) and start from there.